src/asInput/index.jsx

```diff
@@ -109,43 +109,14 @@ const asInput = (WrappedComponent, inputType = undefined, labelFirst = true) =>
       const descriptionId = `description-${this.state.id}`;
       const desc = {};
 
-      const hasDangerTheme = this.hasDangerTheme();
-
       desc.error = (
-        <div
-          className={classNames(
-            styles['form-control-feedback'],
```

Comment:
`form-control-feedback` is a bootstrap class that adds a bit more formatting for us, curious on why it was removed? https://v4-alpha.getbootstrap.com/components/forms/ (search for the class name)

Reply:
Looking more into those docs, we weren't using it correctly is my guess on the answer -- so new question, should we be?

Reply:
I think `form-control-feedback` was removed in Bootstrap v4.0. It also looks like Bootstrap only supports green or red validation themes now (`is-valid` and `is-invalid` classes): https://getbootstrap.com/docs/4.0/components/forms/#server-side